docs/release_notes/0.38.0.md

```diff
@@ -0,0 +1,34 @@
+# Release 0.38.0
+
+## Features
+
+- Support sourceSecurityGroupIds for unmanaged nodegroups (#3254)
+- added RoleOnly option for iamserviceaccounts (#3122)
+- Flux v2 support (#3066)
+- Add support for well known policies with IRSA (#3045) 
+
+## Improvements
+
+- Add option to pass kubeconfig path to Flux 2 installer (#3256)
+- always enable log timestamps (#3251)
+- Add deprecation notice to enable repo command (#3206)
+- Detect and log unsupported region error for fully-private clusters (#3186)
+- ignore ResourceNotFoundException error during cluster deletion (#3210)
+- Set/Unset/Get labels on unowned nodegroups (#3168)
+- Add x-kubernetes-group-version-kind to Schema (#3169)
+- Don't install nvidia for inf1 instances, reenable inf1 integration tests (#3162)
+
+## Bug Fixes
+
+- Support secondary CIDR blocks for VPCs (#3232)
+- Fix creating private cluster in some regions (#3228)
+- Fix suspend ASG processes for nodegroups (#3218)
+- Support position name in get iamserviceaccount (#3205)
+- Support parsing YAML Cloudformation templates (#3170)
+- Refactor iamserviceaccounts  (#3135)
+- Do not error for empty get nodegroups if output is yaml or json (#3178)
+
+## Acknowledgments
+Weaveworks would like to sincerely thank:
+ @Legion2, @dbluxo, @dholbach and @tkms0106
```

Comment:
oh wait should dholbach be here? isn't he one of ours?

Reply:
:shrug: he's not on the ignore list

Reply:
🤷‍♀️